src/btree/bt_debug.c

```diff
@@ -960,18 +934,20 @@ __debug_page_col_int(WT_DBG *ds, WT_PAGE *page, uint32_t flags)
 
 	session = ds->session;
 
-	WT_INTL_FOREACH_BEGIN(session, page, ref) {
+	WT_INTL_FOREACH_BEGIN (session, page, ref) {
 		WT_RET(ds->f(ds, "\trecno %" PRIu64 "\n", ref->ref_recno));
 		WT_RET(__debug_ref(ds, ref));
-	} WT_INTL_FOREACH_END;
+	}
+	WT_INTL_FOREACH_END;
 
 	if (LF_ISSET(WT_DEBUG_TREE_WALK))
-		WT_INTL_FOREACH_BEGIN(session, page, ref) {
+		WT_INTL_FOREACH_BEGIN (session, page, ref) {
 			if (ref->state == WT_REF_MEM) {
 				WT_RET(ds->f(ds, "\n"));
 				WT_RET(__debug_page(ds, ref, flags));
 			}
-		} WT_INTL_FOREACH_END;
+		}
+	WT_INTL_FOREACH_END;
```

Comment:
It's because the `if` doesn't have braces. So it interprets the the `WT_INTL_FOREACH_BEGIN` as the only statement.
The solution is just to put braces around this stuff. If we feel strongly about it, I can include these changes in the aforementioned PR.